experimental/devices/inky/inky.go

```diff
@@ -0,0 +1,291 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package inky
+
+import (
+	"encoding/binary"
+	"fmt"
+	"image"
+	"image/color"
+	"log"
+	"time"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/display"
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/conn/spi"
+)
+
+const (
+	// Constants for an Inky pHAT
+	cols = 104
+	rows = 212
+
+	speed      = 488 * physic.KiloHertz
+	spiBits    = 8
+	chunkSize  = 4096
+	spiCommand = gpio.Low
+	spiData    = gpio.High
+)
+
+type Color int
```

Comment:
Please add a comment about the fact that it can be used to specify which model is being used, and also to set the border color.